recipes-connectivity/networkd-dhcp-conf/networkd-dhcp-conf.bb

```diff
@@ -15,8 +15,8 @@ SRC_URI = " \
   "
 PR = "r1"
 
-RDEPENDS_${PN} = "systemd"
-RCONFLICTS_${PN} = "connman"
+REQUIRED_DISTRO_FEATURES_${PN} = "systemd"
+CONFLICT_DISTRO_FEATURES_${PN} = "connman"
```

Comment:
But connman is not a distro feature, is it?

Reply:
I am not sure if I actually understand  a notion of distro feature at all :), but based of what I read here https://www.yoctoproject.org/docs/2.5/ref-manual/ref-manual.html#var-DISTRO_FEATURES and here https://www.yoctoproject.org/docs/2.5/ref-manual/ref-manual.html#ref-features-distro any package or set of packages can be defined and exposed as a distro feature. 
Anyway, I am rolling that line back to the previous version.

Reply:
Yeah, I'm also not sure what is the exact difference, but I think connman just never gets into the DISTRO_FEATURES variable.

Reply:
This was still wrong, see https://github.com/advancedtelematic/meta-updater/pull/771